torch/_inductor/kernel/mm.py

```diff
@@ -1086,6 +1086,10 @@ def tuned_scaled_mm(
     if use_aten_gemm_kernels():
         choices.append(aten_choice)
 
+    # We dont have triton lowerings for the MX variants yet
```

Comment:
proper fix is to add `dot_scaled` based triton kernel.

Not sure if we want to jam into existing triton template our if we want a separate template